### PR TITLE
proofreading

### PR DESCRIPTION
**Pull request by nicklashansen**
two instances of if -> it in torch.nn.modules.batchnorm.py

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit 3aee28d4:


* **5/5** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=3aee28d4f1acb1389c0d7ebf69c23c407002c1c6).

### :detective: 5 new failures recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4402547) binary_linux_conda_2_7_cpu_devtoolset7_test (1/5)
**Step:** "Should run job" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=94654685) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=94654685))
<details>
<summary>
<code>/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found</code>
</summary>

```
+ '[' -z 'conda 2.7 cpu devtoolset7' ']' 
+ '[' -e /home/circleci/workspace/.circleci/scripts/COMMIT_MSG ']' 
+ '[' -n https://github.com/pytorch/pytorch/pull/29797 ']' 
+ [[ pull/29797 != \c\i\-\a\l\l\/* ]] 
+ [[ pull/29797 != \n\i\g\h\t\l\y ]] 
+ [[ pull/29797 != \p\o\s\t\n\i\g\h\t\l\y ]] 
+ '[' -e /home/circleci/workspace/.circleci/scripts/should_run_job.py ']' 
+ python /home/circleci/workspace/.circleci/scripts/should_run_job.py 'conda 2.7 cpu devtoolset7' 
Rejecting conda 2.7 cpu devtoolset7 
+ circleci step halt 
/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4402528) pytorch_linux_xenial_py3_5_build (2/5)
**Step:** "Should run job" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=94654727) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=94654727))
<details>
<summary>
<code>/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found</code>
</summary>

```
+ '[' -z pytorch-linux-xenial-py3.5-build ']' 
+ '[' -e /home/circleci/workspace/.circleci/scripts/COMMIT_MSG ']' 
+ '[' -n https://github.com/pytorch/pytorch/pull/29797 ']' 
+ [[ pull/29797 != \c\i\-\a\l\l\/* ]] 
+ [[ pull/29797 != \n\i\g\h\t\l\y ]] 
+ [[ pull/29797 != \p\o\s\t\n\i\g\h\t\l\y ]] 
+ '[' -e /home/circleci/workspace/.circleci/scripts/should_run_job.py ']' 
+ python /home/circleci/workspace/.circleci/scripts/should_run_job.py pytorch-linux-xenial-py3.5-build 
Rejecting pytorch-linux-xenial-py3.5-build 
+ circleci step halt 
/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4402524) pytorch_paralleltbb_linux_xenial_py3_6_gcc5_4_build (3/5)
**Step:** "Should run job" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=94654728) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=94654728))
<details>
<summary>
<code>/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found</code>
</summary>

```
+ '[' -z pytorch-paralleltbb-linux-xenial-py3.6-gcc5.4-build ']' 
+ '[' -e /home/circleci/workspace/.circleci/scripts/COMMIT_MSG ']' 
+ '[' -n https://github.com/pytorch/pytorch/pull/29797 ']' 
+ [[ pull/29797 != \c\i\-\a\l\l\/* ]] 
+ [[ pull/29797 != \n\i\g\h\t\l\y ]] 
+ [[ pull/29797 != \p\o\s\t\n\i\g\h\t\l\y ]] 
+ '[' -e /home/circleci/workspace/.circleci/scripts/should_run_job.py ']' 
+ python /home/circleci/workspace/.circleci/scripts/should_run_job.py pytorch-paralleltbb-linux-xenial-py3.6-gcc5.4-build 
Rejecting pytorch-paralleltbb-linux-xenial-py3.6-gcc5.4-build 
+ circleci step halt 
/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4402535) pytorch_parallelnative_linux_xenial_py3_6_gcc5_4_build (4/5)
**Step:** "Should run job" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=94654729) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=94654729))
<details>
<summary>
<code>/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found</code>
</summary>

```
+ '[' -z pytorch-parallelnative-linux-xenial-py3.6-gcc5.4-build ']' 
+ '[' -e /home/circleci/workspace/.circleci/scripts/COMMIT_MSG ']' 
+ '[' -n https://github.com/pytorch/pytorch/pull/29797 ']' 
+ [[ pull/29797 != \c\i\-\a\l\l\/* ]] 
+ [[ pull/29797 != \n\i\g\h\t\l\y ]] 
+ [[ pull/29797 != \p\o\s\t\n\i\g\h\t\l\y ]] 
+ '[' -e /home/circleci/workspace/.circleci/scripts/should_run_job.py ']' 
+ python /home/circleci/workspace/.circleci/scripts/should_run_job.py pytorch-parallelnative-linux-xenial-py3.6-gcc5.4-build 
Rejecting pytorch-parallelnative-linux-xenial-py3.6-gcc5.4-build 
+ circleci step halt 
/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4402574) binary_linux_manywheel_3_7m_cu100_devtoolset7_test (5/5)
**Step:** "Should run job" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=94654944) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=94654944))
<details>
<summary>
<code>/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found</code>
</summary>

```
+ '[' -z 'manywheel 3.7m cu100 devtoolset7' ']' 
+ '[' -e /home/circleci/workspace/.circleci/scripts/COMMIT_MSG ']' 
+ '[' -n https://github.com/pytorch/pytorch/pull/29797 ']' 
+ [[ pull/29797 != \c\i\-\a\l\l\/* ]] 
+ [[ pull/29797 != \n\i\g\h\t\l\y ]] 
+ [[ pull/29797 != \p\o\s\t\n\i\g\h\t\l\y ]] 
+ '[' -e /home/circleci/workspace/.circleci/scripts/should_run_job.py ']' 
+ python /home/circleci/workspace/.circleci/scripts/should_run_job.py 'manywheel 3.7m cu100 devtoolset7' 
Rejecting manywheel 3.7m cu100 devtoolset7 
+ circleci step halt 
/home/circleci/workspace/.circleci/scripts/should_run_job.sh: line 25: circleci: command not found 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>
